Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.12.8-alpine3.21
+
+ENV PYTHOUNNBUFFERED 1
```

Comment:
There's a typo in the environment variable name. It should be `PYTHONUNBUFFERED` instead of `PYTHOUNNBUFFERED`. Correcting this will ensure the application runs as expected.